pom.xml

```diff
@@ -86,7 +86,7 @@
     <dependency>
       <groupId>org.apache.sshd</groupId>
       <artifactId>sshd-core</artifactId>
-      <version>1.0.0</version>
+      <version>1.7.0</version>
```

Comment:
Note to self: careful! Needs to be tested manually, not just using `JenkinsRule`, due to `pluginFirstClassLoader`.

Reply:
```console
$ mvn -f jenkinsci/jenkins/war dependency:tree | fgrep sshd
[INFO] +- org.jenkins-ci.modules:sshd:jar:2.6:compile
[INFO] |  +- org.apache.sshd:sshd-core:jar:1.7.0:compile
```

Reply:
or as of 2.138.4

```
[INFO] +- org.jenkins-ci.modules:sshd:jar:2.4:compile
[INFO] |  \- org.apache.sshd:sshd-core:jar:1.7.0:compile
```

Reply:
see also #36